### DISCUSSION TITLE
Wave App Load takes too long

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2202] [Q&A]
Hi, I moved my Wave server and app code to a new server. When I running my wave app using the command wave run app.py directly, When i try to connect from a browser window while typing http://localhost:10101/app it takes time to display and displays a plain white page with the loading message. What could be the issue?Both my server and app are on the same server. I am using wave version 1.0.0, Server is Windows 64 bit

**Comment by mturoci** [comment]
Do you experience the same locally?
